src/module/entities/helper.js

```diff
@@ -6,97 +6,1165 @@
  */
 
 // Dependencies
-
-
-const { v4: uuidv4 } = require('uuid')
-
+const entityTypesHelper = require(MODULES_BASE_PATH + '/entityTypes/helper')
+// const elasticSearch = require(ROOT_PATH + "/generics/helpers/elasticSearch");
+// const userRolesHelper = require(MODULES_BASE_PATH + "/userRoles/helper");
+// const userProfileService = require("../../generics/");
+const entitiesQueries = require(DB_QUERY_BASE_PATH + '/entities')
+// const formService = require(PROJECT_ROOT_DIRECTORY + '/generics/services/form')
 // const entitiesHelper = require(MODULES_BASE_PATH + "/entities/helper")
-const _ = require("lodash")
+const _ = require('lodash')
 // const programUsersQueries = require(DB_QUERY_BASE_PATH + "/programUsers")
 
 /**
-    * UserProjectsHelper
-    * @class
-*/
+ * UserProjectsHelper
+ * @class
+ */
 
 module.exports = class UserProjectsHelper {
+	static processEntityMappingUploadData(mappingData = []) {
```

Comment:
@MallanagoudaB provide function signature, where ever missing , check all the files
sample : https://github.com/shikshalokam/ml-core-service/blob/master/module/entities/helper.js